app/services/claims/sampling/generate_csv_file.rb

```diff
@@ -33,6 +34,7 @@ def call
             claim.school.local_authority_name,
             claim.school.type_of_establishment,
             claim.school.group,
+            claim.provider.name,
```

Comment:
```suggestion
            claim.provider_name,
```

Can we use delegation here?